src/FacadeAccountCreation/FacadeAccountCreation.Core/Services/Organisation/OrganisationService.cs

```diff
@@ -279,4 +279,31 @@ public async Task UpdateOrganisationDetails(
 
         response.EnsureSuccessStatusCode();
     }
+
+    public async Task<List<OrganisationNationModel>> GetOrganisationNationByExternalIdAsync(Guid organisationExternalId)
+    {
+        var url = $"{OrganisationNationUrl}?organisationId={organisationExternalId}";
+
+        try
+        {
+            _logger.LogInformation("Attempting to fetch the nation for an organisation id {organisationExternalId} from the backend", organisationExternalId);
+
+            var response = await _httpClient.GetAsync(url);
+
+            if (response.StatusCode == HttpStatusCode.NoContent) return null;
+
+            response.EnsureSuccessStatusCode();
+
+            return await response.Content.ReadFromJsonWithEnumsAsync<List<OrganisationNationModel>>();
+        }
+        catch (Exception e)
+        {
+            _logger.LogError(e, "Failed to get Organisation nation for Organisation Id: '{organisationExternalId}'", organisationExternalId);
+            throw;
```

Comment:
as you have inplimented NotFound() in your controller,Please consider returning null which means this would result in an expected response rather then 500

Reply:
I have updated this to follow the common pattern in the service for http calls.